schema/GrantRecordWithRequestedOperationParaExample.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
thanks, I rename it to be "GrantRecordExample.json" and I delete the other example, because the requestedOperationParam is required for release 1, so we don't have 2 cases anymore.